internal/source/source.go

```diff
@@ -72,7 +72,7 @@ func getNodeAtLine(fileset *token.FileSet, astFile ast.Node, lineNum int) (ast.N
 			return node, err
 		}
 	}
-	return nil, nil
+	return nil, fmt.Errorf("failed to find expression")
```

Comment:
There is no reason to use fmt.Errorf when your are not formatting

```suggestion
	return nil, errors.New("failed to find expression")
```

Reply:
Fixed 👍